Tests/SwiftDocCTests/Infrastructure/PathHierarchyTests.swift

```diff
@@ -1318,6 +1318,24 @@ class PathHierarchyTests: XCTestCase {
         // This overloaded protocol method should be able to resolve without a suffix at all, since it doesn't conflict with anything
         let overloadedProtocolMethod = try tree.findNode(path: "/ShapeKit/OverloadedProtocol/fourthTestMemberName(test:)", onlyFindSymbols: true)
         XCTAssert(overloadedProtocolMethod.symbol?.identifier.precise.hasSuffix(SymbolGraph.Symbol.overloadGroupIdentifierSuffix) == true)
+
+    }
+
+    func testAmbiguousPathsForOverloadedGroupSymbols() throws {
+        enableFeatureFlag(\.isExperimentalOverloadedSymbolPresentationEnabled)
+        let (_, context) = try testBundleAndContext(named: "OverloadedSymbols")
+        let tree = context.linkResolver.localResolver.pathHierarchy
+        try assertPathRaisesErrorMessage("/ShapeKit/OverloadedProtocol/fourthTestMemberName(test:)-abc123", in: tree, context: context, expectedErrorMessage: """
+        'abc123' isn't a disambiguation for 'fourthTestMemberName(test:)' at '/ShapeKit/OverloadedProtocol'
+        """) { error in
+            XCTAssertEqual(error.solutions, [
+                .init(summary: "Remove '-abc123' to select preferred symbol\n'func fourthTestMemberName(test: String) -> Double\'", replacements: [("", 56, 63)]),
```

Comment:
Should `PathHierarchyBasedLinkResolver.fullName(of:in:)` have special behavior for overload groups so that it returns the simplified declaration instead of whichever declarations happened to be copied for the underlying symbol that represents the overload group?

Reply:
I adjusted the logic at the call site, and just called `node.name` instead. Let me know if you think we should push this down into `full name(of:in:)`. Probably we should not, since the simplified declaration is not the full name. They seem like separate concepts.